src/core/include/openvino/opsets/opset.hpp

```diff
@@ -25,6 +25,7 @@ class OPENVINO_API OpSet {
 
 public:
     OpSet() = default;
+    OpSet(const std::string&);
```

Comment:
A public header like this one should contain argument names in my opinion.

Reply:
Done, thanks!